src/main/java/uk/gov/hmcts/tools/RepositoryOptimiser.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.tools;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.Project;
+import org.gradle.api.artifacts.repositories.ArtifactRepository;
+
+import javax.inject.Inject;
+import java.util.Arrays;
+import java.util.Collections;
+
+
+public class RepositoryOptimiser extends DefaultTask {
```

Comment:
I don't think extending defaulttask is needed here, you could just have the static method.